drivers/volume/ocp/ocp.go

```diff
@@ -0,0 +1,102 @@
+package ocp
+
+import (
+	"fmt"
+	"github.com/libopenstorage/openstorage/api"
+	torpedovolume "github.com/portworx/torpedo/drivers/volume"
+	"github.com/portworx/torpedo/drivers/volume/portworx"
+	"github.com/portworx/torpedo/drivers/volume/portworx/schedops"
+	"github.com/portworx/torpedo/pkg/errors"
+	"github.com/portworx/torpedo/pkg/log"
+)
+
+const (
+	OcpDriverName = "ocp"
+	// OcpServiceName is the name of the ocp storage driver implementation
+	OcpServiceName = ""
+)
+
+// Provisioners types of supported provisioners
+var provisionersForOcp = map[torpedovolume.StorageProvisionerType]torpedovolume.StorageProvisionerType{
+	OcpDriverName: "openshift-storage.rbd.csi.ceph.com",
```

Comment:
Change this to `rbd` and `cephfs` instead of `OcpDriverName` and create folder structure for spec accordingly